server/src/lib.rs

```diff
@@ -138,32 +138,34 @@ pub async fn run(max_buffered_commands: u32, max_concurrent_queries: u32) {
 pub fn initialize_app(genesis_config: GenesisConfig) -> Application<dependency::Dependency> {
     let mut app = dependency::create(&genesis_config);
 
-    let (genesis_changes, table_changes, evm_storage_changes) = {
-        #[cfg(test)]
-        {
-            moved_genesis_image::load()
-        }
-        #[cfg(not(test))]
-        {
-            moved_genesis::build(
-                &moved_genesis::MovedVm::new(&genesis_config),
-                &genesis_config,
-                &app.evm_storage,
-            )
-        }
-    };
-    moved_genesis::apply(
-        genesis_changes,
-        table_changes,
-        evm_storage_changes,
-        &genesis_config,
-        &mut app.state,
-        &mut app.evm_storage,
-    );
+    if app.block_queries.latest(&app.storage).unwrap().is_none() {
```

Comment:
Feels good how it's pretty much a line change because everything was in place already.